CHANGELOG.md

```diff
@@ -1,3 +1,107 @@
+# 0.0.106 - 2022-04-01
+
+## API Updates
+ * Minimum supported rust version (MSRV) is now 1.41.1 (#1310).
+ * Lightning feature `option_scid_alias` is now supported and may be negotiated
+   when opening a channel with a peer. It can be configured via
+   `ChannelHandshakeConfig::negotiate_scid_privacy` and is off by default but
+   will be on by default in the future (#1351).
+ * `OpenChannelRequest` now has a `channel_type` field indicating the features
+   the channel will operate with and should be used to filter channels with
+   incompatible features (#1351).
```

Comment:
"incompatible" makes it sound like users have to reject features that LDK doesn't support, which isn't true. Maybe "undesirable (see Serialization Compatibility Section)"?